website/static/js/pages/project-settings-page.js

```diff
@@ -67,14 +67,16 @@ $(document).ready(function() {
     }
     var disableCategory = !window.contextVars.node.parentExists;
     // need check because node category doesn't exist for registrations
-    if ($('#nodeCategorySettings').length) {
-        var categorySettingsVM = new ProjectSettings.NodeCategorySettings(
-            window.contextVars.node.category,
-            categories,
-            window.contextVars.node.urls.update,
-            disableCategory
-        );
-        $osf.applyBindings(categorySettingsVM, $('#nodeCategorySettings')[0]);
+    if ($('#projectSettings').length) {
+        var projectSettingsVM = new ProjectSettings.ProjectSettings( {
+            currentTitle: ctx.node.title,
+            currentDescription: ctx.node.description,
+            category: ctx.node.category,
+            categories: categories,
```

Comment:
@abought I'm actually not sure where 'categories' is being used. I know it's populated in project-settings-page.js but I don't see it used in projectSettings.js. I just left that code untouched